design/cli-install-changes.md

```diff
@@ -0,0 +1,426 @@
+      --restic-pod-mem-request string              memory request for restic pod. A value of "0" is treated as unbounded. Optional. (default "0")
+```
+
+...`backup-location create`
+```
+      --backup-location-config mapStringString     configuration to use for the backup storage location. Format is key1=value1,key2=value2
+      --bucket string                              name of the object storage bucket where backups should be stored
+      --prefix string                              prefix under which all Velero data should be stored within the bucket. Optional.
+```
+
+...`snapshot-location create`
+```
+      --snapshot-location-config mapStringString   configuration to use for the volume snapshot location. Format is key1=value1,key2=value2
+```
+
+...both `backup-location create` and `snapshot-location create`
+```
+      --provider string                            provider name for backup and volume storage 
+```
+
+...`plugin`
+```
+      --plugins stringArray                        Plugin container images to install into the Velero Deployment
+      --sa-annotations mapStringString             annotations to add to the Velero ServiceAccount. Add iam.gke.io/gcp-service-account=[GSA_NAME]@[PROJECT_NAME].iam.gserviceaccount.com for workload identity. Optional. Format is key1=value1,key2=value2
+      --no-secret                                  flag indicating if a secret should be created. Must be used as confirmation if --secret-file is not provided. Optional.
+      --secret-file string  (renamed `credentials-file`)   file containing credentials for backup and volume provider. If not specified, --no-secret must be used for confirmation. Optional.
+```
+
+Flags to delete:
+```
+      --no-default-backup-location                 flag indicating if a default backup location should be created. Must be used as confirmation if --bucket or --provider are not provided. Optional.
+      --use-volume-snapshots                       whether or not to create snapshot location automatically. Set to false if you do not plan to create volume snapshots via a storage provider. (default true)
+      --wait                                       wait for Velero deployment to be ready. Optional.
+```
+
+##### Velero Server
+`velero server (RENAMED velero config server)` 
+ 
+`velero server --default-backup-storage-location (DEPRECATED)` changed to `velero backup-location set --default` 
+
+`velero server --default-volume-snapshot-locations (DEPRECATED)` changed to `velero snapshot-location set --default`
+
+`velero server --default-restic-prune-frequency (DEPRECATED)` changed to `velero config restic set --default-prune-frequency` 
+
+`velero server --restic-timeout  (DEPRECATED)` changed to `velero config restic set timeout`
+
+`velero server --use-restic  (DEPRECATED)` see `velero config restic`
+
+All other `velero server` flags changed to under `velero config server`.
```

Comment:
Agreed, I think we probably by-and-large don't want **any** changes to the `velero server` command, since the server still needs to run the same way, with the only possible exceptions being `--default-backup-storage-location` and --default-volume-snapshot-locations` if we're moving to storing those as annotations.